src/platforms/dialogflow/directives/facebook.ts

```diff
@@ -0,0 +1,85 @@
+import * as _ from "lodash";
+
+import { IDirective } from "../../../directives";
+import { ITransition } from "../../../StateMachine";
+import { IVoxaEvent } from "../../../VoxaEvent";
+import { IVoxaReply } from "../../../VoxaReply";
+import { DialogFlowEvent } from "../DialogFlowEvent";
+import { DialogFlowReply } from "../DialogFlowReply";
+
+export class FacebookAccountLink implements IDirective {
```

Comment:
can you check if this works when using the `reply` way of specifying a response?